### PR TITLE
Italian translation of the XMPP Newsletter August 2024

### PR DESCRIPTION
**Pull request by nicfab**
File: `content/blog/newsletter/2024-09-05-newsletter.it.md`

### CODE REVIEW

File: content/blog/newsletter/2024-09-05-newsletter.it.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+L'XSF è stato accettato come [organizzazione ospitante della GSoC nel 2024](https://wiki.xmpp.org/web/Google_Summer_of_Code_2024)! Questi progetti XMPP hanno ricevuto uno slot e hanno iniziato la loro attività di codifica:
+
+- [Monal](https://monal-im.org/)
+    - [Modern Monal onboarding e Media gallery (175 ore, medio)](https://wiki.xmpp.org/web/Gsoc2024/Monal/Media_Gallery)
+        - [Post del blog 1](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-1-xmpp-standards-foundation-da781ac95560)
+        - [Post sul blog 2](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-2-midterm-evaluations-xmpp-standards-foundation-3be8b27dc653)
+
+
+<center>
```
Comment (Echolon) [inline content/blog/newsletter/2024-09-05-newsletter.it.md:21]: @cal0pteryx that works well?

File: content/blog/newsletter/2024-09-05-newsletter.it.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+-   [Istanza di Lemmy (non ufficiale)](https://slrpnk.net/c/xmpp)
+-   [Reddit (non ufficiale)](https://www.reddit.com/r/xmpp/)
+-   [Pagina Facebook XMPP (non ufficiale)](https://www.facebook.com/jabber)
+
+
+<br>
+
+<center>
+    
+<div id="rectangle" style="padding: 5px; border: 2px solid blue; border-radius: 10px;">
```
Comment (Echolon) [inline content/blog/newsletter/2024-09-05-newsletter.it.md:201]: Is that as it should be?

File: content/blog/newsletter/2024-09-05-newsletter.it.md (hunk 1 above)
Comment (cal0pteryx) [reply content/blog/newsletter/2024-09-05-newsletter.it.md:21]: No, this is HTML, which won't be rendered by Hugo. We use shortcodes for this on xmpp.org

File: content/blog/newsletter/2024-09-05-newsletter.it.md (hunk 2 above)
Comment (cal0pteryx) [reply content/blog/newsletter/2024-09-05-newsletter.it.md:201]: No, see my other comment

**Comment by cal0pteryx** [comment]
Added in d29e10ead1fd0d45b60d4caf3012e878043dcb84

**Pull request closed without merging**